codejail/tests/test_jail_code.py

```diff
@@ -613,8 +590,8 @@ def test_crash_cpython(self):
         # the `new` module has been removed in Python 3, and replaced with the
         # `types` module. This test code needs to be refactored to support
         # both.
-        if six.PY3:
-            raise SkipTest('Not supported in Python 3 yet')
+        raise SkipTest('Not supported in Python 3 yet')
+        # pylint: disable=unreachable
```

Comment:
We should probably do the refactoring mentioned in the comment above, now that we only need to support the news `types` module version.

Reply:
I think we can do that change later on as I don't have enough context or domain knowledge about the tests right now to make this change.